backend/app/services/helpOffers/update.ts

```diff
@@ -0,0 +1,6 @@
+import * as helpOffersRepo from "../../repositories/helpOffers/update"
+
+export const update = async(user_id:string, updatedHelpOffer:any) => {
+    const helpOffer = await helpOffersRepo.update(user_id, updatedHelpOffer)
```

Comment:
we can validate here like basic check whether u helpOffer is not undefined

Reply:
Yeye I haven't really done any of that, havedone